src/components/Header.js

```diff
@@ -5,12 +5,20 @@ const Header = ({ isHomepage }) => {
   const homepageClass = isHomepage ? 'homepage-header' : ''
   return (
     <header className={`site-header ${homepageClass}`}>
-      <Link to="/"><div className="logo">Example Site</div></Link>
+      <Link to="/">
+        <div className="logo">Example Site</div>
+      </Link>
       <nav>
         <ul>
-          <li><Link to="/">Home</Link></li>
-          <li><Link to="/about">About</Link></li>
-          <li><Link to="/more-info">More Info</Link></li>
```

Comment:
I honestly don't like using auto-formatting tools like Prettier. Most of the time it's alright, but there are certain cases like this where I think it looks better and more readable before.